dotnet/src/SemanticKernel/TemplateEngine/PromptTemplateEngine.cs

```diff
@@ -65,17 +65,17 @@ public async Task<string> RenderAsync(string templateText, SKContext context)
     public async Task<string> RenderAsync(IList<Block> blocks, SKContext context)
     {
         this._log.LogTrace("Rendering list of {0} blocks", blocks.Count);
-        var result = new StringBuilder();
+        var tasks = new List<Task<string>>();
         foreach (var block in blocks)
         {
             switch (block)
             {
                 case ITextRendering staticBlock:
-                    result.Append(staticBlock.Render(context.Variables));
+                    tasks.Add(Task.FromResult<string>(staticBlock.Render(context.Variables)));
```

Comment:
On second glace, I think you can drop the `<string>` part, as it'll be implicitly typed by the compiler.